### PR TITLE
The portal catalog will not try to index itself anymore

### LINKED ISSUE #3312: Plone 6: rebuilding the catalog does not work
Rebuilding the catalog using the ZMI does not work

See: https://community.plone.org/t/upgrade-plone-6-sites-dx-site-root/14303/16?u=alert

### PR DESCRIPTION
**Pull request by ale-rt**
Fixes #3312

### CODE REVIEW

**Comment by ale-rt** [comment]
@jenkins-plone-org please run jobs

**Review by mauritsvanrees — APPROVED** [review]
LGTM. I tested it, and it helps.

Should be back ported to CMFPlone 5.2, because there the catalog also indexes itself.

**Review by tisto — APPROVED** [review]

File: news/3312.bugfix [hunk 1]
```diff
@@ -0,0 +1 @@
+The portal catalog will not try to index itself anymore
```
Comment (tisto) [inline news/3312.bugfix:1]: @ale-rt you forgot to put your name here. :)

**Comment by ale-rt** [comment]
Plone 5.2 PR is here: #3316

**Comment by ale-rt** [comment]
@jenkins-plone-org please run jobs

**Pull request merged**